language/vm/src/file_format.rs

```diff
@@ -338,8 +338,17 @@ pub struct FieldInstantiation {
     pub type_parameters: SignatureIndex,
 }
 
-/// A `StructDefinition` is a type definition. It either indicates it is native or
-// defines all the user-specified fields declared on the type.
+/// A `FriendDeclaration` encapsulates information about a friend.
+#[derive(Clone, Debug, Hash, Eq, PartialEq)]
+#[cfg_attr(any(test, feature = "fuzzing"), derive(Arbitrary))]
+#[cfg_attr(any(test, feature = "fuzzing"), proptest(no_params))]
+pub struct FriendDeclaration {
+    /// Handle index to the friend module
+    pub module: ModuleHandleIndex,
+}
+
```

Comment:
We should remove this struct as the encapsulation is not needed

Reply:
`FriendDeclaration` dropped.